docs/guides/upgrading.rst

```diff
@@ -50,6 +53,13 @@ New API for working with file uploads
  * ``elgg_get_uploaded_files()`` - returns an array of Symfony uploaded file objects
  * ``ElggFile::acceptUploadedFile()`` - moves an uploaded file to Elgg's filestore
 
+New API for manipulating images
+-------------------------------
+
+New image manipulation service implements a more efficient approach to cropping and resizing images.
```

Comment:
It's uses less memory as it doesn't have to keep all the temporary data/raw bytes in memory. Plus we can switch to imagick.